drivers/sensor/mpu9250/ak8963.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
```

Comment:
this header is not self-contained:

```suggestion
#include <stdint.h>

#include <device.h>
#include <drivers/sensor.h>
#include <sys/util_macro.h>
```

Reply:
Fixed